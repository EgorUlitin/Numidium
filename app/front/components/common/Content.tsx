import * as React from 'react'

import { css } from 'emotion'


interface Props {
    children?: any
}

export default (props: Props) =>
    <div className={s.content}>
        {props.children}
    </div>


const s = {
    content: css`
        padding: 24px;
        min-height: 360px;
    `,
}
