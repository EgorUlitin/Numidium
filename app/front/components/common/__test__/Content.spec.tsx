import * as React from 'react'
import * as renderer from 'react-test-renderer'

import Content from '../Content'


test('`Content` renders correctly', () => {
    const tree = renderer
        .create(
            <Content>
                <p>First s</p>
            </Content>
        )
        .toJSON()

    expect(tree).toMatchSnapshot()
})
