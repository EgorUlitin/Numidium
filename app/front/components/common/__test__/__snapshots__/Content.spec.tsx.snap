// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Content\` renders correctly 1`] = `
<div
  className="css-1nkmqjx"
>
  <p>
    First s
  </p>
</div>
`;
